data/battle_scripts_1.s

```diff
@@ -9926,6 +9926,16 @@ BattleScript_BerserkGeneRet_End:
 	removeitem BS_SCRIPTING
 	end3
 
+BattleScript_BoosterEnergyEnd2::
```

Comment:
Is there a reason this is not just `BattleScript_BoosterEnergyEnd`?

Reply:
The script ends with `end2`. Tbh I don't know the difference. Though it seems there is no consistency at all. Some scripts don't even have the suffix even when they should